### PR TITLE
Cultist shades made by non-cultists will now drop their HuD, and have a warning message.

### LINKED ISSUE #21805: non-cult shades
Shades created by security shouldn't be able to see cultists on the hud that's just unfair.

### PR DESCRIPTION
**Pull request by ShiftyRail**
[role] [balance] [discussion]

This essentially was how it was in Cult II. Minus of course the ability to make security-aligned constructs.
Waiting for an headmin ruling (haha) I feel this is the mos reasonable course of action.

The current course of action with the rule that the shade remains a cultist would
1) Penalise security for trying to spare a cultist
2) Be very confusing for the poor culted person

Forbidding non-cultists to use stones would
1) Be gay
2) Remove a wizard feature

Closes #21805

### CODE REVIEW

File: code/datums/gamemode/factions/legacy_cult/soulstone.dm [hunk 1]
```diff
@@ -329,10 +329,9 @@
 		dir = NORTH
 		update_icon()
 	user.update_inv_hands()
-	to_chat(shadeMob, "Your soul has been captured! You are now bound to [user.name]'s will, help them suceed in their goals at all costs.")
+	to_chat(shadeMob, "<span class='notice'>Your soul has been captured! You are now bound to [user.name]'s will, help them suceed in their goals at all costs.</span>")
```
Comment (Kurfursten) [inline code/datums/gamemode/factions/legacy_cult/soulstone.dm]: >suceed

Comment (ShiftyRail) [reply code/datums/gamemode/factions/legacy_cult/soulstone.dm]: Not me that typed this !

**Comment by DeityLink** [comment]
fine by me.

**Comment by jknpj** [comment]
>mommi made a new Balance tag

🤔

**Comment by DeityLink** [comment]
That being said, if the shade is allowed to exit the soulstone, or enter a construct shell, and was a cultist, They should be fully loyal to the cult again.

**Comment by DeityLink** [comment]
also
>adds a decult() proc
>doesn't add a recult() proc for when the shade gets back to the cult's side

Just check whenever a shade leave the stone whether their mind whether they have a role in the cult faction's members.

**Comment by ShiftyRail** [comment]
You will take my security-aligned army of juggers from my cold, dead hands

**Comment by DeityLink** [comment]
That I will.

This is ridiculous, non-cultists can already use stones.

**Comment by Caboozles** [comment]
Jugger vs Jugger battles was like the best part of cult 2.0

It’s almost offensive you can’t do that in cult 3. The sheer act of looting a shell alone should lead to a punishment against the cultists.

**Comment by DeityLink** [comment]
That reduces the incentive of having a cult base at all, which was a main draw of cult 3, and gives sec-"we have silicons on our side"-curity a buff they really don't need.

Not to mention that non-cult juggers will still be unable to guess who's on their side. which was already an endless source of ahelps the last time jugger vs jugger was a thing @Caboozles

**Comment by DeityLink** [comment]
Not to mention that non-cult juggers will still be unable to guess who's on their side.

**Comment by DeityLink** [comment]
Anyway this PR still doesn't account for the situation where the shade gets back on the cult's side. Which kinda tells a lot about your motivations.

also

`Forbidding non-cultists to use stones would    1. Be gay     2. Remove a wizard feature`

That's not the case since the issue only occurs when capturing an already cultist shade, don't put misleading stuff in the OP

**Comment by DeityLink** [comment]
Lastly, I've got a decent compromise to suggest you @ShiftyRail . That security trying to use a soul gem automatically turns it into a soul stone shard. This way:
* Security can once again use constructs if they get their hands on occult loot
* The cult gets to keep the exclusivity to their new upgraded versions
* Juggernauts fighting each others can easily identify who's on which side since they now look different

Non-cultists already lack the knowledge to properly put a soul gem in a cult blade, it makes sense that they'd fail to properly put a whole gem into a construct shell.

**Comment by Kammerjunk** [comment]
>That's not the case since the issue only occurs when capturing an already cultist shade, don't put misleading stuff in the OP

now Shifty is a big boy who can speak for himself but I'm pretty sure it was referring to the big old closed-doors #metaclub channel talk about shades

**Comment by DeityLink** [comment]
He might be, but that's the problem, people who aren't in the #metaclub and who wander into this thread might think that my position/the current state is that non-cultists cannot use soulstones, which isn't true.

**Comment by ShiftyRail** [comment]
The problem I was trying to solve was "non-cult aligned shades still get the HuD and can out the cult", with the offered solution - "ban them" - deemed unacceptable by several admins - kammer, veers, chaz, and myself.

I saw this as a code issue I offered to solve by taking their HuD and cult speak.
Then was brought the topic of security aligned juggers, on which corporatism may have gotten in the way of the discussion.

The comprise seems fine to me.

**Comment by DeityLink** [comment]
wait who offered to ban them? let me rewind the channel a bit.

**Comment by ShiftyRail** [comment]
```
[16:12:06]ADMIN: Announce: Kurfurst/(Tess Dummy) : Official adminstrative policy is that if you can see HUD icons indicating who is a faction member, you're part of that faction, don't follow other priorities to out them. Someone who isn't at work will be logging on shortly to handle discipline.
```

**Comment by DeityLink** [comment]
ok that was Kurfurst. I was busy at that moment so I didn't have time to join in on the discussion but I agree that players shouldn't be banned for something that isn't made obvious.
If the compromise I posted above is fine with you and we have a temporary recult() proc (until the ability to just mute/unmute roles is implemented) then I agree.

**Comment by Kurfursten** [comment]
It requires a serious lack of common sense or willful ignorance to believe simultaneously
1. You are not loyal to the cult
2. For some reason you get to see all cult members with a floating icon

How do you get yourself into this situation and not believe it's an exploit - that it fundamentally ruins the game mode when it happens? Like, for instance, if someone discovered one day that being defibrillated caused them to see all the AntagHUD icons and they went around validing all the tots and vamps.

Big yikes for everybody involved in that debacle (including Shifty as a sec department member)

**Comment by ShiftyRail** [comment]
Using shades as a non-cultists is something that was always a thing, and is intended (or at least was in Cult II).
There are several special provisions in the code for when the holder is a cultist ; if it was not meant for cultist use at all, then, there would only be one check :

```dm
if (!iscultist(user))
      return
```

The only bad thing is that shades still have their HuD. Which this PR adresses.

We shouldn't ban for a code oversight, especially when shading people was common - and intended - previously.

**Comment by Kammerjunk** [comment]
banhunger is a big mood

**Comment by Kurfursten** [comment]
If nothing else, this makes the situation better than it was before. There may yet be a better solution but until that day, this one has enough support to be merged.

**Pull request merged**